covid_moonshot_ml/utils.py

```diff
@@ -185,6 +186,8 @@ def train(
         Number of epochs to train for
     device : torch.device
         Where to run the training
+    loss_fn : cml.nn.MSELoss
+        Loss function
```

Comment:
some more details on the type of loss fn would be helpful here

Reply:
done in [6a84868](https://github.com/choderalab/covid-moonshot-ml/pull/33/commits/6a84868ad01de3f85c2305059ec20e1874292ec1)